atlasdb-config/src/main/java/com/palantir/atlasdb/http/ProtocolAwareExceptionMapper.java

```diff
@@ -37,6 +39,8 @@
 
 abstract class ProtocolAwareExceptionMapper<E extends Exception> implements ExceptionMapper<E> {
     private static final SafeLogger log = SafeLoggerFactory.get(ProtocolAwareExceptionMapper.class);
+    private static final RateLimiter LEGACY_OR_UNKNOWN_RATE_LIMITER =
```

Comment:
I think @gsheasby extracted a class for this sort of logging rate limiting

Reply:
Yes, please use `RateLimitedLogger` here.